[2021-08-04 14:42:08,700] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_A 2021-08-04T14:41:57.958228+00:00 [queued]>
[2021-08-04 14:42:08,720] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_A 2021-08-04T14:41:57.958228+00:00 [queued]>
[2021-08-04 14:42:08,720] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 14:42:08,720] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-04 14:42:08,720] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 14:42:08,738] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): training_model_A> on 2021-08-04T14:41:57.958228+00:00
[2021-08-04 14:42:08,744] {standard_task_runner.py:52} INFO - Started process 1470 to run task
[2021-08-04 14:42:08,749] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'training_model_A', '2021-08-04T14:41:57.958228+00:00', '--job-id', '2913', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpfl5sni0t', '--error-file', '/tmp/tmpeicywz9y']
[2021-08-04 14:42:08,750] {standard_task_runner.py:77} INFO - Job 2913: Subtask training_model_A
[2021-08-04 14:42:08,803] {logging_mixin.py:104} INFO - Running <TaskInstance: my_dag.training_model_A 2021-08-04T14:41:57.958228+00:00 [running]> on host 04479ea62028
[2021-08-04 14:42:08,864] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=training_model_A
AIRFLOW_CTX_EXECUTION_DATE=2021-08-04T14:41:57.958228+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-04T14:41:57.958228+00:00
[2021-08-04 14:42:08,865] {python.py:151} INFO - Done. Returned value was: 7
[2021-08-04 14:42:08,892] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=training_model_A, execution_date=20210804T144157, start_date=20210804T144208, end_date=20210804T144208
[2021-08-04 14:42:08,935] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-04 14:42:08,962] {local_task_job.py:149} INFO - Task exited with return code 0
